File: Assignment_43/Program3.c
/* Write a recursive program which accept string from user and count number of Small Characters.
Input : HElloWOrlD
Output : 5
*/

#include <stdio.h>

int Small(char *str)
{
   static int iCount = 0;

   if(*str != '\0')
   {
       if((*str >= 'a') && (*str <= 'z'))
       {
           iCount++;
       }
       str++;

       Small(str);
   }
       return iCount;
   
   
}

int main()
{
    int iRet = 0;
    char arr[20];

    printf("Enter string : ");
    scanf("%[^\n]",arr);

    iRet = Small(arr);

    printf("Number of Small characters are :%d\n",iRet);

    return 0;
}

File: Assignment_46/Program3.java
/* Write Java Program which accept number of rows and number of columns from user and display below pattern.

Input :  iRow = 3      iCol = 5
Output:  A  A  A  A  A
         B  B  B  B  B
         C  C  C  C  C
*/

import java.util.*;

class Pattern
{
    public void Pattern(int iRow,int iCol)
    {
        int i = 0, j = 0;
        char ch = 'A';

        for(i = 1; i <= iRow; i++)
        {
            for(j = 1; j <= iCol; j++)
            {
                System.out.print(ch+"\t");
            }
            System.out.println();
            ch++;
        }
    }
}

class Program3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number of Rows : ");
        int iValue1 = sobj.nextInt();

        System.out.println("Enter the number of Columns : ");
        int iValue2 = sobj.nextInt();

        Pattern pobj = new Pattern();
        pobj.Pattern(iValue1,iValue2);
    }
}